storage/cloud-client/snippets_test.py

```diff
@@ -287,3 +294,81 @@ def test_bucket_lifecycle_management(test_bucket, capsys):
     out, _ = capsys.readouterr()
     assert "[]" in out
     assert len(list(bucket.lifecycle_rules)) == 0
+
+
+def test_create_bucket_class_location():
+    bucket = storage_create_bucket_class_location.create_bucket_class_location(
+        "storage-snippets-test"
+    )
+
+    assert bucket.location == 'US'
+    assert bucket.storage_class == "COLDLINE"
+    bucket.delete(force=True)
+
+
+def test_bucket_delete_default_kms_key(test_bucket, capsys):
+    storage_bucket_delete_default_kms_key.bucket_delete_default_kms_key(
+        test_bucket.name
+    )
+
+    out, _ = capsys.readouterr()
+    assert test_bucket.default_kms_key_name is None
+    assert test_bucket.name in out
+
+
+def test_get_service_account(capsys):
+    storage_get_service_account.get_service_account()
+    out, _ = capsys.readouterr()
+
+    assert "@gs-project-accounts.iam.gserviceaccount.com" in out
+
+
+def test_download_public_file(test_blob):
+    storage_make_public.make_blob_public(test_blob.bucket.name, test_blob.name)
+    with tempfile.NamedTemporaryFile() as dest_file:
+        storage_download_public_file.download_public_file(
+            test_blob.bucket.name, test_blob.name, dest_file.name
+        )
+
+        assert dest_file.read()
```

Comment:
Does this actually verify this sample works?  Won't this just ready an empty file if not? 

The preferred pattern for samples is [Arrange, Act, Assert](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#arrange-act-assert):

1. Arrange - prepare the request
2. Act - send request, and receive response
3. Assert - Print response (to show the user how to interact with it) and assert the response occurred when the sample was run

It's an explict non-goal for our samples tests to test the service or product. Instead they should be that the _sample_ ran successfully, and should verify the output of the sample in a reasonable way. Thus, most sample tests should be in the pattern:

1. Run the sample
2. Check the output of the sample